sdk/storage/azfile/sas/query_params_test.go

```diff
@@ -132,7 +132,7 @@ func TestIPRange_String(t *testing.T) {
 
 func TestSAS(t *testing.T) {
 	// Note: This is a totally invalid fake SAS, this is just testing our ability to parse different query parameters on a SAS
-	const sas = "sv=2019-12-12&sr=b&st=2111-01-09T01:42:34.936Z&se=2222-03-09T01:42:34.936Z&sp=rw&sip=168.1.5.60-168.1.5.70&spr=https,http&si=myIdentifier&ss=bf&srt=s&rscc=cc&rscd=cd&rsce=ce&rscl=cl&rsct=ct&sig=clNxbtnkKSHw7f3KMEVVc4agaszoRFdbZr%2FWBmPNsrw%3D"
```

Comment:
Is it possible to add a test where the SAS url is used to create/add files?

Reply:
Done